python/lsst/meas/base/applyApCorr.py

```diff
@@ -291,6 +300,12 @@ def run(self, catalog, apCorrMap):
                 if self.config.doFlagApCorrFailures:
                     source.set(apCorrInfo.fluxFlagKey, oldFluxFlagState)
 
+                # Log a message if it has been a while since the last log.
+                if (currentTime := time.time()) > nextLogTime:
+                    self.log.verbose("Aperture corrections applied to %d sources out of %d",
+                                     sourceIndex + 1, len(catalog))
```

Comment:
Thanks Tim! I just changed `sourceIndex` to `sourceIndex + 1` to start the counting from 1.